dts/bindings/interrupt-controller/arm,vim.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI Vectored Interrupt Manager
+
+compatible: "arm,vim"
+
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/59191#discussion_r1268291067 , TI Vectored Interrupt Manager is a external interrupt controller(TI specific IP) which is compatible with R5F VIC port.

Reply:
I will update the description to "TI Vectored Interrupt Manager is a external interrupt controller(TI specific IP) which is compatible with R5F VIC port."?